### PR TITLE
Fix obtaining NativeEvent* values

### PR DESCRIPTION
**Pull request by wokalski**

### CODE REVIEW

File: src/RNEvent.re [hunk 1]
```diff
@@ -3,23 +3,23 @@ type t;
 module NativeEvent = {
   type t;
   external changedTouches : t => array (Js.t {..}) = "" [@@bs.get];
-  external identifier : t => int = "" [@@bs.get];
-  external locationX : t => int = "" [@@bs.get];
-  external locationY : t => int = "" [@@bs.get];
-  external pageX : t => int = "" [@@bs.get];
-  external pageY : t => int = "" [@@bs.get];
+  external identifier : t => float = "" [@@bs.get];
```
Comment (saschatimme) [inline src/RNEvent.re]: is the identifier really a `float`?

Comment (wokalski) [inline src/RNEvent.re]: no, it should be int. Search and replace got me!

Comment (saschatimme) [inline src/RNEvent.re]: happens the best!

It's the same for `timestamp`

**Comment by wokalski** [comment]
Fixed. Additionally I changed externals in `rei` to `let`s.

**Pull request merged**